features/user_profile_picture.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Users should be able to set their profile picture
+    As a user
+    I want to set a profile picture
+    So others can see my dog
+
+Background: User already login
+    Given the default layout exist
+    And I am on the RateMyPup home page
+
+    And the following users exist:
+        | username       | email               | password       | password_confirmation| activated |
+        | jeff           | jeff@berkeley.edu   | 12345678       | 12345678             | true      |
+
+Scenario: user sets profile image correctly
+  Given I log in as "jeff"
+  And I am on the "Edit Profile" page
```

Comment:
These steps can be moved to the background as they are repeated for both scenarios.

Reply:
Good idea